activesupport/test/evented_file_update_checker_test.rb

```diff
@@ -36,6 +36,8 @@ def rm_f(files)
   end
 
   test 'notifies forked processes' do
+    skip if defined?(JRUBY_VERSION)
```

Comment:
We have a helper method `jruby_skip` for skipping tests on JRuby. 

We can just that here. `jruby_skip "Forking not available on JRuby" :smile: